platform/pulp/platform/models/progress.py

```diff
@@ -19,6 +19,19 @@
 _logger = logging.getLogger(__name__)
 
 
+def get_current_task_id():
```

Comment:
I'm actually moving this function as part of the conversion of the tasking code to use the new models. What if we let my PR clean up this import and you continue to use it from `pulp.server.async.tasks` ?